red-hat-storage/ocs-ci
# Flexible Scaling on UI

Test Procedure:
https://bugzilla.redhat.com/show_bug.cgi?id=1943280

1.Add a new disk to worker node
2.Add capacity via UI
3.Verify all OSD pods move to Running state
4.check ceph health after add capacity

Signed-off-by: Oded Viner <oviner@redhat.com>
